src/integrationTest/sam.test.ts

```diff
@@ -262,7 +352,7 @@ describe('SAM Integration Tests', async function () {
                 })
 
                 it('invokes and attaches on debug request (F5)', async function () {
-                    setTestTimeout(this.test?.fullTitle(), 60000)
+                    setTestTimeout(this.test?.fullTitle(), 90000)
```

Comment:
Any specific reasons for the time bump?

Reply:
If download takes a long time then yes we need to increase test timeout.